src/components/Calendar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { DatePicker } from 'redux-form-material-ui';
```

Comment:
This library is really helpful in terms of dealing with redux-form, I understand the push to drop this, but i would suggest that we keep it as long as we are using Material-UI for now.